packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -232,6 +239,8 @@ class Auth {
 
         this.defaultDnt = config.defaultDnt
 
+        this.refreshTokenTTL = config.refreshTokenTTL || 90 * 24 * 60 * 60 // 90 days in seconds
```

Comment:
This could arguably fall back to the authoritative claim from the SLAS jwt here if the override here is not set:

https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2077/files#diff-4f3bb7407698d0a81a28ff00462f1589482fa8e984b6b19622286c5f6db36bb1R415-R418

Reply:
Yes! let's remove the numbers here and use the `refresh_token_expires_in` SLAS response 👍